pyfr/plugins/solnwriter.py

```diff
@@ -0,0 +1,40 @@
+# -*- coding: utf-8 -*-
+
+from collections import OrderedDict
+
+from pyfr.h5writer import H5Writer
+from pyfr.inifile import Inifile
+from pyfr.plugins.base import BasePlugin
+
+
+class SolnWriterPlugin(BasePlugin):
+    name = 'solnwriter'
+    systems = ['*']
+
+    def __init__(self, intg, cfgsect, *args, **kwargs):
```

Comment:
Either specify all of the parameters (including suffix) or just `*args` and `**kwargs` and let the parent constructor handle them for you.